task_sdk/tests/execution_time/test_supervisor.py

```diff
@@ -0,0 +1,150 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from __future__ import annotations
+
+import inspect
+import logging
+import os
+import signal
+import sys
+from unittest.mock import MagicMock
+
+import pytest
+import structlog
+import structlog.testing
+
+import airflow.sdk.api.client
```

Comment:
```suggestion
from airflow.sdk.api import client
```

Reply:
or was it on purpose?

Reply:
Probably on purpose of "`Client` on it's own is a big ambigious what it is"

Reply:
How about?
```suggestion
from airflow.sdk.api import client as sdk_client
```

Reply:
Yeah that works

Reply:
Done